### PR TITLE
Adding cars function

### PR DESCRIPTION
**Pull request by devdummy778**
Added the cars variable, its awesome works great

### CODE REVIEW

**Comment by xnbeats** [comment]
Sorry , can you fix this.

**Comment by xnbeats** [comment]
This need to be fixed

File: cars.js [hunk 1]
```diff
@@ -0,0 +1 @@
+const cars = "This is my cars file";
```
Comment (xnbeats) [inline cars.js:1]: This needs to say CAR.

Comment (xnbeats) [inline cars.js:1]: Starting a review

**Review by xnbeats — CHANGES_REQUESTED** [review]
Please fix this.

File: cars.js (hunk 1 above)
Comment (devdummy778) [reply cars.js:1]: Problem fixed

**Review by xnbeats — APPROVED** [review]
Approved

**Pull request merged**